numba_dppy/compiler.py

```diff
@@ -491,6 +492,9 @@ def _unpack_argument(
 
         device_arrs.append(None)
 
+        if isinstance(ty, USM_NdArrayType):
+            pass
```

Comment:
Let us raise an `NotImplementedError` in this PR. I will clean it up in my next PR.